tests/integration/test_sns.py

```diff
@@ -149,6 +149,69 @@ def test_filter_policy(self):
         num_msgs_2 = len(self.sqs_client.receive_message(QueueUrl=self.queue_url_2, VisibilityTimeout=0)['Messages'])
         self.assertEqual(num_msgs_2, num_msgs_1)
 
+    def test_exists_filter_policy(self):
+        # connect SNS topic to an SQS queue
+        queue_arn = aws_stack.sqs_queue_arn(TEST_QUEUE_NAME_2)
+        filter_policy = {'store': [{'exists': True}]}
+        self.sns_client.subscribe(
```

Comment:
nit: We could put this code into a small util function, to avoid duplication with lines 186-193:
```
    def do_subscribe(filter_policy):
        self.sns_client.subscribe(
            TopicArn=self.topic_arn,
            Protocol='sqs',
            Endpoint=queue_arn,
            Attributes={
                'FilterPolicy': json.dumps(filter_policy)
            }
        )
    ...
    do_subscribefilter_policy)
    ...
    do_subscribe(filter_policy)
```